src/encoder/encoderfdkaac.h

```diff
@@ -0,0 +1,218 @@
+// encoderfdkaac.h
+// Created on Aug 15 2017 by Palakis
+
+#ifndef ENCODER_ENCODERFDKAAC_H
+#define ENCODER_ENCODERFDKAAC_H
+
+#include <QLibrary>
+#include <QString>
+
+#include "encoder/encoder.h"
+#include "util/fifo.h"
+
+class EncoderFdkAac: public Encoder {
+  public:
+    EncoderFdkAac(EncoderCallback* pCallback, QString pFormat);
+    virtual ~EncoderFdkAac();
+
+    int initEncoder(int samplerate, QString errorMessage) override;
+    void encodeBuffer(const CSAMPLE *samples, const int sampleCount) override;
+    void updateMetaData(const QString& artist, const QString& title, const QString& album) override;
+    void flush() override;
+    void setEncoderSettings(const EncoderSettings& settings) override;
+
+  private:
+    QString buttWindowsFdkAac();
+
+    // libfdk-aac common AOTs
+    static const int AOT_AAC_LC = 2; // AAC-LC
```

Comment:
please add a comment where these values come from. 
We normally initialize all variables in the C++ file.